neo/test/README.txt

```diff
@@ -2,18 +2,17 @@ To run all tests:
 
     $ python -m unittest discover
 
-If you have nose installed:
+If you have pytest installed:
 
-    $ nosetests
+    $ pytest
 
 
 To run tests from an individual file:
 
     $ python test_analogsignal.py
```

Comment:
Is this python or pytest? to run an individual file now? Or both?

Reply:
We should review this (not now, but soon). Originally, each test file could be run directly with python (since they just used unittest). I'm not sure if this is still the case.